src/Codeception/Module/Phalcon34x.php

```diff
@@ -0,0 +1,674 @@
+<?php
+namespace Codeception\Module;
+
+use Phalcon\Di;
+use PDOException;
+use Phalcon\Mvc\Url;
+use Phalcon\DiInterface;
+use Phalcon\Di\Injectable;
+use Codeception\TestInterface;
+use Codeception\Configuration;
+use Codeception\Lib\Framework;
+use Phalcon\Mvc\RouterInterface;
+use Phalcon\Mvc\Model as PhalconModel;
+use Phalcon\Mvc\Router\RouteInterface;
+use Codeception\Util\ReflectionHelper;
+use Codeception\Exception\ModuleException;
+use Codeception\Lib\Interfaces\ActiveRecord;
+use Codeception\Lib\Interfaces\PartedModule;
+use Codeception\Exception\ModuleConfigException;
+use Codeception\Lib\Connector\Phalcon as PhalconConnector;
```

Comment:
It looks like your module is using the old connector here.
If module code is exactly the same, it would be better to update Phalcon module to select connector dynamically, based on some version check or existance of some class, method or property that was introduced or removed in 3.4.0.

Reply:
fair enough.

Reply:
If the only difference is in MemorySession class, than please create it dynamically and keep using the same connector too,

Reply:
@Naktibalda I simplified everything to just replace the session based on the internal phalcon version + I extended the session class so only the necessary changes are reflected.

Reply:
didn't even read your comment but came to the same conclusion. :)